### PR TITLE
Update dependancies

### PR DESCRIPTION
**Pull request by dandahle**
Merge Rohin's changes into develop for testing

### CODE REVIEW

File: controllers/api.js [hunk 1]
```diff
@@ -503,13 +503,14 @@ getDocumentPlanning: function (req, res, next) {
...
-                
+                    project: { $exists: false },
+
                 },
                 // Updates
                 {
-                    // $set: {name: value}
-                    $set: {project: {status: "project"}},
+                    // $set: {"project.status": "project"},
+                    $set: {"project": {status: "project"}},
```
Comment (rohinadalja) [inline controllers/api.js]: Line 489, Line 512: These lines will need to be changed as this will overwrite the entire project field when triggered.

**Comment by dandahle** [comment]
Yikes! Don’t want that to happen. Is this part of the vulnerability fixes or what you found when looking into the revising of applications resetting project info? Thanks for looking into this, Rohin. ==> Dan

**Comment by rohinadalja** [comment]
@dandahle  This is why project associated data is being overwritten when going back and forth with regards to updating the project status.  (Fixes https://github.com/dandahle/Catalyst-AppVetting/issues/111).

**Comment by dandahle** [comment]
After testing the merged code on AWS develop instance we found that the Application status can no longer be changed and saved. The status change appears in the current Vetting window, but when returning to the Vetting Summary or refreshing the page the Application status is unchanged. 

Restoring the update-dependencies branch and Reverting the merge. Rohin will evaluate when he has time.

**Pull request merged**